src/main/scala/inox/ast/ExprOps.scala

```diff
@@ -141,8 +141,8 @@ trait ExprOps extends GenTreeOps {
       case StringConcat(StringLiteral(""), b) => b
       case StringConcat(b, StringLiteral("")) => b
       case StringConcat(StringLiteral(a), StringLiteral(b)) => StringLiteral(a + b)
-      case StringLength(StringLiteral(a)) => IntLiteral(a.length)
-      case SubString(StringLiteral(a), IntLiteral(start), IntLiteral(end)) =>
+      case StringLength(StringLiteral(a)) => Int32Literal(a.length)
+      case SubString(StringLiteral(a), Int32Literal(start), Int32Literal(end)) =>
```

Comment:
This was actually a but in Inox: `StringLength` and `SubString` are defined for Integers and not bitvectors. Could you please fix this in this PR? Thanks!